tools/dbTools.py

```diff
@@ -56,54 +48,25 @@ def _execute(self, query, params=()):
             )
             raise
 
-    def _check_db(self):
-        """Create the tables needed by the web scraper if they don't exists."""
-        self._execute(
-            """
-            CREATE TABLE IF NOT EXISTS article
-            (
-                id SERIAL PRIMARY KEY,
-                url VARCHAR(255),
-                file_hash VARCHAR(32),
-                scrap_again BOOLEAN DEFAULT FALSE,
-                timestamp TIMESTAMP DEFAULT CURRENT_TIMESTAMP
-            );
-            """
-        )
-        self._execute(
-            """
-            CREATE TABLE IF NOT EXISTS spiders
-            (
-                id SERIAL PRIMARY KEY,
-                name VARCHAR(64),
-                uuid VARCHAR(64),
-                status VARCHAR(255),
-                end_time TIMESTAMP,
-                start_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP
-            );
-            """
-        )
-        self.connection.commit()
-
     def reset_scraped(self):
-        """Set all the articles `scrap_again` attribute to 1, forcing the web
-        scraper to download and analyse them again.
+        """Set all the publications `scrape_again` attribute to 1, forcing the
+        web scraper to download and analyse them again.
```

Comment:
what are the way a publication might be flagged for scrape_again?

also at the moment it is all or nothing, right? have you ever used it?

Reply:
I flag them manually at the moment, but the idea is to be able to flag them on certain criteria